### PR TITLE
*: cut 2.0.0-beta.3

### PR DESCRIPTION
**Pull request by fabxc**
@Gouthamve @grobie

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,17 @@
-## v2.0.0-beta.2 / 2017-08-18
+## v2.0.0-beta.2 / 2017-09-12
```
Comment (grobie) [inline CHANGELOG.md]: beta.3

Comment (Gouthamve) [inline CHANGELOG.md]: Umm, overwriting beta2 here?

Comment (fabxc) [reply CHANGELOG.md]: Yes, forgot the beta number though.

Comment (fabxc) [reply CHANGELOG.md]: That's how we've done it for all non-stable releases so far.

**Pull request merged**